python/_eos.cc

```diff
@@ -498,11 +498,28 @@ BOOST_PYTHON_MODULE(_eos)
         ;
 
     // ObservableCache
-    class_<ObservableCache>("ObservableCache", no_init)
+    class_<ObservableCache>("ObservableCache", R"(
+        Provides a cache for the efficient evaluation of observables.
+    )", init<const Parameters &>())
         .def("__iter__", range(&ObservableCache::begin, &ObservableCache::end))
-        .def("__getitem__", &ObservableCache::operator[])
-        .def("add", &ObservableCache::add)
-        .def("update", &ObservableCache::update)
+        .def("__getitem__", &ObservableCache::operator[], R"(
+            Access the cached value of an observable.
+
+            :param handle: The handle of the observable.
+            :type handle: int
+        )", args("handle"))
+        .def("add", &ObservableCache::add, R"(
+            Adds an existing observable to the cache.
```

Comment:
For consistency I would either remove the `s` here or add it in the previous definition.